aiida/cmdline/params/options/main.py

```diff
@@ -204,6 +205,22 @@ def set_log_level(_ctx, _param, value):
     '-G', '--group', 'group', type=types.GroupParamType(), help='A single group identified by its ID, UUID or label.'
 )
 
+SOURCE_GROUP = OverridableOption(
```

Comment:
Do we really need these to be reusable? They are currently only used once. The docstrings are also not very useful. They are generic, but when used for `move-nodes` the help will be not so useful. Would simply use the following in the command directly:
```python
@options.GROUP(
    '-s',
    '--source-group',
    required=True,
    help='The group whose nodes to move.'
)
@options.GROUP(
    '-t',
    '--target-group',
    required=True,
    help='The group to which the move to.
)
```

Reply:
Right, I remembered I tried this myself, but got the following error:

```
verdi group move-nodes --help
Traceback (most recent call last):
  File "/home/mbercx/.virtualenvs/aiida-core/bin/verdi", line 5, in <module>
    from aiida.cmdline.commands.cmd_verdi import verdi
  File "/home/mbercx/envs/aiida-core/code/aiida-core/aiida/cmdline/commands/__init__.py", line 14, in <module>
    from aiida.cmdline.commands import (
  File "/home/mbercx/envs/aiida-core/code/aiida-core/aiida/cmdline/commands/cmd_group.py", line 88, in <module>
    @options.GROUP(
TypeError: __call__() takes 1 positional argument but 3 were given
```

I guess you can only override the `OverridableOption` args using keyword arguments, and I'm not sure what the keywords are for the option flags?

Reply:
Yeah, or course, forgot about that, but that makes sense. The idea of a reusable option is to guarantee it has the same flag everywhere so allowing to change it doesn't make sense. No matter though, just use the suggestion still but simply use  the plain `@click.option` and add `type=GroupParameterType`.

Reply:
Makes sense, thanks! 👍